src/loops/simple_problems_with_repitition/medium/q1/MainTest.java

```diff
@@ -0,0 +1,61 @@
+package loops.simple_problems_with_repitition.medium.q1;
+
+import global.variables.*;
+import global.variables.clauses.*;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import java.util.stream.Stream;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import global.BaseTest;
+import global.exceptions.InvalidClauseException;
+import global.tools.TestOption;
+
+
+public class MainTest extends BaseTest {
+    // Parsons
+
+    public Clause[] testSentence() {
+
+        TestOption.isInputTest = true;
+        TestOption.defaultInput = "10";
+
+        return new Clause[] {
+            new StringLiteral("Enter a number : "),
+            new NewLine(),
+            new IntegerLiteral("numberOutput"),
+            new StringLiteral(" in binary is "),
+            new PlaceHolder()
+        };
+    }
+
+    public void runMain() {
+        InTheBin.main(new String[0]);
+    }
+
+    public String convertBinary(int n) {
```

Comment:
In this case, we should just hardcode input/ouput pairs into the `inputProvider` instead of using this helper method